Readme.md

```diff
@@ -1,17 +1,18 @@
 # oneContainer API
 
-OneContainer-API is a platform to enable unified APIs for containerized services and backends in multiple segments like AI, Database, and Media.
```

Comment:
can u pls update the readme with the missing docs some got deleted..as seen here